contents/blog/product-health-metrics.md

```diff
@@ -0,0 +1,125 @@
+### Feature usage
+
+Having users is great, but you want them to actually use your product. Measuring feature usage helps you do this. This is tracking the usage of parts of your product, such as creating, analyzing, or sharing content. Feature usage is a more specific health metric than new user growth or churn rate. It shows the health of specific features, which enables it to be more actionable.
+
+A way to do this is with the lifecycle insight. It enables you to see new, returning, and resurrected usage of the feature as well as how many users have gone dormant.
+
+![Lifecycle](../images/blog/product-health-metrics/lifecycle.png)
+
+Feature usage helps you understand the health of individual parts of your product. Like the body, if you heal the "sick" parts, the whole is healthier. Feature usage improves prioritization by aligning development time with usage.
+
+### Session duration
+
+Depending on the product, session duration, or its average, are useful product health metrics. It is a general measurement of user engagement. It shows if users are spending significant time in your product. If this changed significantly, something shifted in your app to cause users to spend less time there.
+
+For products reliant on power users, the extremes of session duration can be more important. As average users are not your target users, the average session duration can be misleading. Improving power user session duration shows there is high demand for your product, people are reliant on it, and that can lead to more success.
+
+For utilities, seeing average session duration _go down_ can be a good sign. This is because your product is becoming more efficient at doing what users want it to do. Saving people time is worth money, and if this is your goal, then making sure your health metrics align helps make it happen.
+
+![Session duration](../images/blog/product-health-metrics/duration.png)
+
+## How to choose the right product health metrics
+
+Picking the right product health metric goes back to what makes a good one. It must be something you want to:
+
+- improve slowly, but a dramatic change would cause concern and intervention.
+- monitor in the short run (daily, weekly) and improve in the long run.
+- know and track because it makes a difference in the quality of the product and experience for users.
+
+As examples, let's go over some types of products and look at what a good health metric might be for them:
+
+- **Subscription B2C:** New user growth, churn rate and cohort retention, extreme session duration, daily and weekly active users.
+- **Ad-based B2C:** Average session duration, DAU/MAU intensity, daily and weekly active users.
+- **Marketplaces:** Feature usage (listing, buying), supplier and buyer lifecycle (check both reoccur and return), new supply growth.
+- **B2B SaaS:** New organization growth, feature usage, organization churn rate.
+- **B2B Usage/User-based:** Feature usage, average session duration, daily and weekly active users.
```

Comment:
These are good, but they could do with context about why they're good for that kind of product. A sentence on each would suffice, I think.